src/NLog/Layouts/Typed/Layout.cs

```diff
@@ -145,7 +145,7 @@ private bool TryConvertTo(object raw, out T value, T defaultValue)
             }
             catch (Exception e)
             {
-                InternalLogger.Debug(e, "Conversion to type {0} failed. Value: {1}", Type, raw.ToString().Left(1000));
+                InternalLogger.Debug(e, "Conversion to type {0} failed. Value: '{1}'", Type, raw.ToString().Left(1000));
```

Comment:
Maybe have conditional `ToString()` or can raw never be null ?

Reply:
null check of raw is in the beginning of the method. :)